internal/controller/policy_subreconciler.go

```diff
@@ -279,6 +280,180 @@ func isPodReady(pod corev1.Pod) bool {
 	return false
 }
 
+func findAdmissionPoliciesForConfigMap(object client.Object) []reconcile.Request {
+	configMap, ok := object.(*corev1.ConfigMap)
+	if !ok {
+		return []reconcile.Request{}
+	}
+	if _, isKubewardenConfigmap := configMap.Labels[constants.PolicyServerLabelKey]; !isKubewardenConfigmap {
+		return []reconcile.Request{}
+	}
+	policyMap, err := getPolicyMapFromConfigMap(configMap)
+	if err != nil {
+		return []reconcile.Request{}
+	}
+	return policyMap.toAdmissionPolicyReconcileRequests()
+}
+
+func findClusterAdmissionPoliciesForConfigMap(object client.Object) []reconcile.Request {
```

Comment:
Quibble: maybe the `find*()` functions here in policy_subreconciler.go could be better named as follows (with matching godocs):
- `findClusterPoliciesForConfigMap()` (applies to both ClusterAdmissionPolicies and ClusterAdmissionPolicyGroups)
- `findPoliciesForConfigMap()` (applies to both AdmissionPolicies and SdmissionPolicyGroups).

Same for the rest of `find*()` functions here. This would make it easier to understand the reuse of code in the separate *controller.go, if I understood it correctly.